### DISCUSSION TITLE
Pesquisa sobre comunidades de tecnologia

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#939] [Dicas]
Fala, galera!

Estou fazendo uma pesquisa sobre as comunidades de tecnologia no Brasil. 

Quem puder contribuir, é só 3 minutinhos e já responde tudo! :grin: 

[Responda este formulário](http://bit.ly/pesquisa-sobre-comunidades)

A pesquisa ficará aberta até o final deste mês e depois eu vou fazer uma landing para exibir os resultados e também vou deixar o link para baixar todo o consolidado de respostas para quem quiser utilizar para gerar outras estatísticas sobre as comunidades.

**Comment by luizbills** [comment]
@woliveiras link quebrado.

**Comment by woliveiras** [comment]
Valeu, mano!

Atualizado
